client/src/components/navigations/MainStackNavigator/ProfileModal.tsx

```diff
@@ -468,10 +476,11 @@ const ModalContent: FC<ModalContentProps> = ({modalState, hideModal}) => {
           modalLayout={modalLayout}
           isStatusMessageExpanded={isStatusMessageExpanded}
           handleAnim={handleAnim}
-          showFriendAddedMessage={showFriendAddedMessage}
-          addFriendInFlight={addFriendInFlight}
         />
       )}
+      <View style={{zIndex: 101}}>
+        <Snackbar testID="profile-snackbar" theme={theme} ref={snackbarRef} />
+      </View>
```

Comment:
I've asked in #470 to put this in `RootStack` for reusability.

https://github.com/dooboolab/hackatalk/pull/470#discussion_r694929377

I think you might want to talk to @woosanggyu who is going to work on this :)

Reply:
#470 is merged, and this PR is ready to be merged :)